plugins/optimization-detective/tests/test-class-od-link-collection.php

```diff
@@ -346,6 +363,9 @@ public function test_add_link( array $links_args, string $expected_html, string
 		}
 
 		$collection = new OD_Link_Collection();
+
+		$this->assertNull( $collection->get_response_header() );
```

Comment:
This call was added to improve test coverage and it exposed error in PHP 7.2/7.3 which is fixed in 921c2d311fad884c814e43a4bba1729becff3c8c